src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/Engine/TreeData.StructuredTrivia.cs

```diff
@@ -17,13 +19,13 @@ private class StructuredTrivia : TreeData
             private readonly TreeData _treeData;
 
             public StructuredTrivia(SyntaxTrivia trivia, int initialColumn)
-                : base(trivia.GetStructure())
+                : base(trivia.GetStructure()!)
```

Comment:
GetRequiredStructure()?

Reply:
Perhaps. There is an assertion a line down which is weird but covers it.